client/coral-embed-stream/src/graphql/index.js

```diff
@@ -122,19 +122,43 @@ const extension = {
     `,
   },
   mutations: {
-    IgnoreUser: () => ({
-
-      // TODO: don't rely on refetching.
-      refetchQueries: [
-        'EmbedQuery', 'EmbedStreamProfileQuery',
-      ],
+    IgnoreUser: ({variables}) => ({
+      updateQueries: {
+        EmbedQuery: (previousData, {mutationResult}) => {
+          const ignoredUserId = variables.id;
+          const response = mutationResult.data.ignoreUser;
+          if (ignoredUserId && !response.errors) {
+            const updated = update(previousData, {me: {ignoredUsers: {$push: [{
+              id: ignoredUserId,
+              __typename: 'User',
+            }]}}});
+            return updated;
+          }
+          return previousData;
+        }
+      }
     }),
-    StopIgnoringUser: () => ({
-
-      // TODO: don't rely on refetching.
+    StopIgnoringUser: ({variables}) => ({
       refetchQueries: [
-        'EmbedQuery', 'EmbedStreamProfileQuery',
+        'EmbedQuery',
```

Comment:
I think the `EmbedQuery` refetch here is not needed. Whenever we change to the `comments` tab, a refetch is triggered automatically.